packages/core/src/builder.class.ts

```diff
@@ -2211,6 +2211,13 @@ export class Builder {
     return observable;
   }
 
+  requestUrl(
```

Comment:
Worth adding a comment about why this is needed

Reply:
done